code/modules/mining/equipment/explorer_gear.dm

```diff
@@ -185,3 +198,48 @@
 	icon_state = "exo"
 	item_state = "exo"
 	resistance_flags = FIRE_PROOF
+
+/obj/item/clothing/suit/hooded/explorer/exo/standard
+	hoodtype = /obj/item/clothing/head/hooded/explorer/exo/standard
+
+/obj/item/clothing/head/hooded/explorer/exo/standard
+
+/obj/item/clothing/suit/hooded/explorer/exo/standard/Initialize()
+	. = ..()
+	AddComponent(/datum/component/armor_plate)
+
+/obj/item/clothing/head/hooded/explorer/exo/standard/Initialize()
+	. = ..()
+	AddComponent(/datum/component/armor_plate)
+    
+
+/obj/item/clothing/suit/space/heavy_miner
+	name = "H.M.S. suit"
+	desc = "Heavy Mining Space Suit is a very armored suit intended for mining and space exploration."
+	icon_state = "mining_heavy"
+	item_state = "mining_heavy"
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+	resistance_flags = FIRE_PROOF | LAVA_PROOF | ACID_PROOF | GOLIATH_RESISTANCE
+	mutantrace_variation = STYLE_DIGITIGRADE
+	slowdown = 1.5
+	armor = list("melee" = 70, "bullet" = 40, "laser" = 30, "energy" = 30, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
```

Comment:
??????????????????????????????????????? no we already have mining hardsuits....

Reply:
The hell, 100% rad resistance? The CE's suit doesn't even have that much.